misk-aws/src/test/kotlin/misk/jobqueue/sqs/DockerSqs.kt

```diff
@@ -85,7 +86,7 @@ internal object DockerSqs : ExternalDependency {
       try {
         client.getQueueUrl("does not exist")
       } catch (e: Exception) {
-        if (e is QueueDoesNotExistException) {
+        if (e is QueueDoesNotExistException || (e is AmazonSQSException && e.statusCode == 400)) {
```

Comment:
My tests were failing locally because the SQS API was not throwing `QueueDoesNotExistException`. Instead it threw `AmazonSQSException` with an error message "AWS.SimpleQueueService.NonExistentQueue; ...<more words>". The error code was not set to something useful. Should we keep this workaround?